src/Nav.js

```diff
@@ -0,0 +1,18 @@
+// #4 This is our new Nav component
+import './TodoListItem.module.css';
+
+const Nav = () => {
+	return (
+		<div>
+			<br />
+        <div class='nav' style={{ paddingBottom: '10px', float: 'center', backgroundColor: 'grey'}} >
```

Comment:
In React/JSX, you use className instead of class. Here's a little article on why: https://www.letsreact.org/why-does-react-use-classname-instead-of-a-class-attribute/#:~:text=The%20only%20reason%20it%20uses,className%20rather%20than%20class%20attribute.